src/proposal/repost.rs

```diff
@@ -0,0 +1,43 @@
+use near_sdk::serde_json::json;
+use near_sdk::{env, AccountId, Promise};
+
+use crate::{get_proposal_description, social_db::social_db_contract, Proposal};
+
+fn repost_internal(proposal: Proposal, contract_address: AccountId) -> near_sdk::serde_json::Value {
+    let proposal_link = format!("/devgovgigs.near/widget/Proposal?id={}", proposal.id);
+    let title = proposal.snapshot.body.clone().latest_version().name;
+
+    let desc = get_proposal_description(proposal.snapshot.body.clone());
+
+    let text = format!(
+        "@{author} [Created the proposal on DevHub]({post_link})\n{title}{desc}",
```

Comment:
This formatting is misleading when we open it in the feed: https://near.org/near/widget/ProfilePage?accountId=devgovgigs.near

<img width="937" alt="image" src="https://github.com/NEAR-DevHub/neardevhub-contract/assets/304265/e9ad4077-7d4f-41b1-99b1-48fc90101468">

It looks like DevHub endorses the content unless you read really carefully.

I suggest using the following text:

```suggestion
        "There is a new proposal on DevHub from @{author}: “{title}“\n> {summary}\n__Read the full proposal [here]({post_link})__”,
```

Use `summary` instead of the `description` of the proposal.